frontend/device/kobo/device.lua

```diff
@@ -877,16 +942,20 @@ function Kobo:resume()
     if not re then
         logger.err('write error: ', err_msg, err_code)
     end
+]]
 
     -- HACK: wait a bit (0.1 sec) for the kernel to catch up
     util.usleep(100000)
     -- cf. #1862, I can reliably break IR touch input on resume...
     -- cf. also #1943 for the rationale behind applying this workaorund in every case...
-    f = io.open("/sys/devices/virtual/input/input1/neocmd", "we")
+    writeToSys("a", "/sys/devices/virtual/input/input1/neocmd")
+--[[
+    local f = io.open("/sys/devices/virtual/input/input1/neocmd", "we")
     if f ~= nil then
         f:write("a\n")
         f:close()
     end
+]]
```

Comment:
Should this stay commented out?
(No idea what it is/was for, @NiLuJe may know ?)

Reply:
Not necessary for standby, necessary for true suspend though.

(Haven't looked at the context here, but if we don't touch `extended-state`, leaving this out is okay, as grid is still powered on anyway).